src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -368,6 +367,23 @@ void GcsServer::InitGcsPlacementGroupManager(const GcsInitData &gcs_init_data) {
   rpc_server_.RegisterService(*placement_group_info_service_);
 }
 
+std::string GcsServer::StorageType() const {
+  if (RayConfig::instance().gcs_storage() == "memory") {
+    if (!config_.redis_address.empty()) {
+      RAY_LOG(INFO) << "Using external Redis for KV storage: " << config_.redis_address;
+      return "redis";
+    }
+    return "memory";
```

Comment:
also print info for memory

Reply:
IIUC, this is already logged here: `RAY_LOG(INFO) << "GCS Storage is set to " << storage_type_;`